schema/Dockerfile

```diff
@@ -7,8 +7,16 @@ RUN git clone https://github.com/pelias/schema.git /code/pelias/schema
 # change working dir
 WORKDIR /code/pelias/schema
 
+# consume the build variables
+ARG REVISION=production
+
+# switch to desired revision
+RUN git checkout $REVISION
+
 # install npm dependencies
 RUN npm install
 
 # run tests
 RUN npm test
+
+CMD ["npm", "run", "create_index"]
```

Comment:
we should only use `CMD` to start long-running scripts, not for one-off tooling scripts such as this.